org.munadi.Munadi.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id": "org.munadi.Munadi",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "command": "munadi",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc", 
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-config/autostart:create",
+        "--talk-name=org.freedesktop.Notifications"
+    ],
+    "modules": [
+        {
+            "name": "munadi",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm644 org.munadi.Munadi.desktop /app/share/applications/org.munadi.Munadi.desktop",
+                "install -Dm644 org.munadi.Munadi.appdata.xml /app/share/appdata/org.munadi.Munadi.appdata.xml",
+                "tar xf org.munadi.Munadi.icons.tar.gz -C /app/share/",
+                "mv src/* .",
+                "qmake munadi.pro",
+                "make -j4",
+                "install -D munadi /app/bin/munadi"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.com/munadi/munadi.git",
+                    "tag": "17.02",
+                    "commit": "316da28372423cb80a0739ab9dca91c777467f0d"
+                },
+                {
+                    "type": "file",
+                    "path": "org.munadi.Munadi.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "org.munadi.Munadi.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "org.munadi.Munadi.icons.tar.gz"
```

Comment:
What is this for? If these icons aren't part of upstream repo, where did you get them from?

Reply:
they could be under upstream, does it matter?
I generated them from 480x480 PNG source, author of app here.